modular_citadel/code/modules/research/designs/medical_designs.dm

```diff
@@ -0,0 +1,58 @@
+///////////////////////////////////////////
+/////////Dedibrillators Disks//////////////
+///////////////////////////////////////////
+
+/datum/design/comp_defib
+	name = "compact defibrillator"
+	desc = "A belt-equipped defibrillator that can be rapidly deployed"
+	id = "comp_defib"
+	build_type = PROTOLATHE
+	build_path = /obj/item/defibrillator/compact
+	materials = list(MAT_METAL=46000, MAT_GLASS = 28000, MAT_GOLD = 26000, MAT_SILVER = 26000, MAT_TITANIUM = 16000)
+	construction_time = 100
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/defib_heal
+	name = "Defibrillartor Healing disk"
+	desc = "A disk alowing for grater amounts of healing"
+	id = "defib_heal"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL=16000, MAT_GLASS = 18000, MAT_GOLD = 6000, MAT_SILVER = 6000)
+	build_path = /obj/item/disk/medical/defib_heal
+	construction_time = 10
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/defib_shock
+	name = "Defibrillartor Anit-Shock Disk"
+	desc = "A disk that helps agains shocking anyone, other then the intented target"
+	id = "defib_shock"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL=16000, MAT_GLASS = 18000, MAT_GOLD = 6000, MAT_SILVER = 6000)
+	build_path = /obj/item/disk/medical/defib_shock
+	construction_time = 10
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/defib_decay
+	name = "Defibrillartor Body-Decay exstender Disk"
+	desc = "A disk that helps defibrillators revive the longer decayed dead"
+	id = "defib_decay"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL=16000, MAT_GLASS = 18000, MAT_GOLD = 16000, MAT_SILVER = 6000, MAT_TITANIUM = 2000)
+	build_path = /obj/item/disk/medical/defib_decay
+	construction_time = 10
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/defib_speed
+	name = "Defibrllartor Pre-Primer Disk"
```

Comment:
This is the only other upgrade that would be fine, also you can just say 'Primer'